packages/plugins/integration-make-com-ui/src/lib/components/make-com-connections/make-com-connections.component.html

```
@@ -0,0 +1,86 @@
+<div class="connections-container">
+  <div class="d-flex justify-content-between align-items-center mb-3">
+    <h6>{{ 'INTEGRATIONS.MAKE_COM_PAGE.CONNECTIONS' | translate }}</h6>
+    <button nbButton size="small" status="basic" (click)="refresh()" [disabled]="loading">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)